### PR TITLE
NEP-10: Invoice-Token-Standard (NRC-10)

### PR DESCRIPTION
**Pull request by liuyong5653**
## Checklist:

- [x] I have read the **NEP Guideline** document.
- [x] I have checked my NEP Draft is ready.

### CODE REVIEW

**Comment by arisac** [comment]
Review Checklist for NEP Editors

`this checklist is for new NEP Draft`

**Check pr NEPs repo**
- [ ] new NEP is placed in /NEPS/nep-title.md
- [ ] assets is placed in /NEPS/assets/NEP-N/asset-name.ext
- [ ] no modifications to other files

**Check NEP formats**
- [ ] header
- [ ] all contents in English

Ask the applicant to make modifications to meet criterions above.

**Assign NEP Number**
- [ ] use pr number if no related issue had opened before
- [ ] change Title and NEP number in document
- [ ] rename nep-title.md to nep-number.md
- [ ] rename assets directory NEP-N to nep-number

**Arrange NEP Board Review**
tbd

**Comment by arisac** [comment]
@liuyong5653  you can use NEP number 10 (#10)

please rename your NEP document to `/NEPS/nep-10.md`

and update header in your NEP document

**Comment by liuyong5653** [comment]
nep-6.md done

File: NEPS/nep-10.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# NEP 10: Invoice Token Standard
+| Item | Description |
+|:-|:-|
+| NEP | 10 |
+| Title | Invoice Token Standard |
+| Author | [Yong Liu](mailto:liuyong5653@163.com), [Evan Liu](mailto:evanliuchina@gmail.com) |
+| Discussions to | https://github.com/newtonproject/NEPs/issues/10 |
+| Status | Draft |
+| Type | Token Template |
```
Comment (evanliuchina) [inline NEPS/nep-10.md:9]: Token Template => NRC

File: NEPS/nep-10.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
+# NEP 10: Invoice Token Standard
```
Comment (evanliuchina) [inline NEPS/nep-10.md:1]: # NEP-10: Invoice Token Standard (NRC-10)

File: NEPS/nep-10.md (hunk 1 above)
Comment (liuyong5653) [reply NEPS/nep-10.md]: done

File: NEPS/nep-10.md (hunk 2 above)
Comment (liuyong5653) [reply NEPS/nep-10.md]: done

**Comment by arisac** [comment]
NEP Board Review https://github.com/newtonproject/NEPs/commit/47586911ba9ecadc2e9acf8903eb0fc3a8b3c6c9

**Pull request merged**